sections/Navbar.jsx

```diff
@@ -38,32 +49,22 @@ export const Navbar = ({
             <PumpImg src={PumpLogo} alt="Pump Logo" width={80} height={50} />
           </a>
         </Link>
-        {isHidden ? (
-          <Icon
-            src={Close}
-            alt="Cross icon"
-            width={30}
-            height={30}
-            onClick={handleClick}
-          />
-        ) : (
-          <Icon
-            src={Open}
-            alt="Hamburger icon"
-            width={30}
-            height={30}
-            onClick={handleClick}
-          />
-        )}
+        <Icon
+          src={isHidden ? Close : Open}
+          alt={isHidden ? "Cross icon" : "Hamburger icon"}
+          width={30}
+          height={30}
+          onClick={handleClick}
+        />
       </LogoContainer>
       <Container isHidden={isHidden}>
         <ResourcesContainer>
-          {RESOURCES.map((resource, index) => (
+          {RESOURCES.map((resource) => (
             <Resource key={resource}>
-              <Link href={`/${PATHS[index]}`}>
+              <Link href={`/${PATH_NAMES[resource]}`}>
```

Comment:
I would pull this out into a variable so it isn't stated twice for repetition now that I think about it